### ISSUE TITLE
Results on CIFAR10 with ResNet34

### ISSUE THREAD

**Issue by kasteric** [Hzzone/ProPos#8]
Hi, thanks for your inspiring work! I have managed to reproduce cifar10 on ResNet18, but I encountered some issues reproducing the results on cifar10 using ResNet34 since my reproduced result with ResNet34 is only 90.4% ACC. Could u please provide the config file for training cifar10 on resnet34? Many thanks!

**Comment by Hzzone** [comment]
```yml
batch_size: 64
num_devices: 4
momentum_base: 0.996
momentum_increase: true
dataset: cifar10
eval_metric:
  - nmi
  - acc
  - ari
whole_dataset: true
encoder_name: bigresnet34
epochs: 1000
feat_dim: 256
hidden_size: 4096
img_size: 32
lambda_predictor_lr: 10
learning_rate: 0.05
learning_eta_min: 0.
reassign: 1
save_freq: 1
save_checkpoints: true
shuffling_bn: true
symmetric: true
temperature: 0.5
use_gaussian_blur: false
warmup_epochs: 50
weight_decay: 0.0005
dist: true
v2: true
use_copy: true

model_name: propos
cluster_loss_weight: 0.1
latent_std: 0.001

wandb: true
project_name: 'clustering_archives'
entity: 'zzhuang'
```
This config should achieve 93%~94% accuracy.
